roles/atomic_containers_list/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# vim: set ft=ansible:
+#
+# list containers and save to json
+#
+- name: List images with --json
+  command: atomic containers list --json
+  register: acl
+
+- name: Save JSON to variable
+  set_fact:
+    acl_json: "{{ acl.stdout | to_json | from_json }}"
```

Comment:
Is it necessary to use both filters?  Seems like you should just be able to use `from_json`?

Reply:
I don't need both.  I'm not quite sure why I did that.